Documentation/energy-output.rst

```diff
@@ -0,0 +1,53 @@
+Energy output
+============
+
+Introduction
+------------
+
+The energy output computes the energy of the simulation. It is divided into multiple parts:
+
+- Energy in the water layer (= acoustic medium).
+    - Gravitational energy
```

Comment:
would be nice to have the formula of each of these energies in the doc...

Reply:
I can provide the code for the energies that I added.

Reply:
Yes, good idea!